scripts/set-disk-partitions

```diff
@@ -12,8 +12,14 @@ fi
 
 PARTITION_COUNT=$(grep $(echo $DEVICE | cut -d '/' -f3) /proc/partitions | wc -l)
 if [ "$PARTITION_COUNT" -gt "1" ]; then
-    echo "Device already partitioned!!! Please use a new device" 1>&2
-    exit 1
+    echo "Device already partitioned!"
+    echo "Checking to see if it is mounted"
+    if [ "$(df |grep ${DEVICE}|wc -l)" -gt "0" ]; then
```

Comment:
Might be safer to check /proc/mounts as `df` can hang in some situations.  This statement can just become `if grep -q "${DEVICE}" /proc/mounts; then`